packages/react-vapor/src/components/tab/Tab.tsx

```diff
@@ -34,6 +37,7 @@ export class Tab extends React.Component<ITabProps, any> {
 
         return (
             <div className={className} onClick={this.handleSelect}>
+                {!!this.props.svgName && <Svg svgName={this.props.svgName} svgClass={this.props.svgClass} />}

```

Comment:
Should we allow any children instead?

It could be useful if we want a different tooltip on the svg for instance

Reply:
Good point! It will be a more flexible option that way (so, I should set a prop that is a React.ReactNode instead).